future/src/ct/ct_storage_sqlite.cc

```diff
@@ -230,6 +231,7 @@ bool CtStorageSqlite::save_treestore(const Glib::ustring& file_path, const CtSto
         {
             _open_db(file_path, false);
 
+            _check_db_tables(_pDb);
```

Comment:
no tables, no need

Reply:
Whoops, yeah put that in the wrong place, it does need to go in the other branch of the if statement though right?

Reply:
Wait nvm, if opening an old file `populate_treestore` will be called before anyway